tests/Neo.Network.RPC.Tests/RpcTestCases.json

```diff
@@ -3050,166 +3050,46 @@
-            "gasconsumed": "2031260",
+        "jsonrpc": "2.0",
+        "id": 1,
+        "result": {
             "exception": null,
-            "stack": [],
-            "notifications": [
-              {
-                "contract": "0x668e0c1f9d7b70a99dd9e06eadd4c784d641afbc",
-                "eventname": "Transfer",
-                "state": {
-                  "type": "Array",
-                  "value": [
-                    {
-                      "type": "ByteString",
-                      "value": "CqOHtT6Wt5iaYxQxoFbdH0CgQvY="
-                    },
-                    {
-                      "type": "Any"
-                    },
-                    {
-                      "type": "Integer",
-                      "value": "18083410"
-                    }
-                  ]
-                }
-              },
-              {
-                "contract": "0x668e0c1f9d7b70a99dd9e06eadd4c784d641afbc",
-                "eventname": "Transfer",
-                "state": {
-                  "type": "Array",
-                  "value": [
-                    {
-                      "type": "Any"
-                    },
-                    {
-                      "type": "ByteString",
-                      "value": "z6LDQN4w1uEMToIZiPSxToNRPog="
-                    },
-                    {
-                      "type": "Integer",
-                      "value": "1252390"
-                    }
-                  ]
-                }
-              }
-            ]
-          },
-          {
-            "trigger": "PostPersist",
+            "trigger": "Application",
             "vmstate": "HALT",
-            "gasconsumed": "2031260",
-            "exception": null,
+            "gasconsumed": 997778,
             "stack": [],
             "notifications": [
-              {
-                "contract": "0x668e0c1f9d7b70a99dd9e06eadd4c784d641afbc",
-                "eventname": "Transfer",
-                "state": {
-                  "type": "Array",
-                  "value": [
-                    {
-                      "type": "Any"
-                    },
-                    {
-                      "type": "ByteString",
-                      "value": "z6LDQN4w1uEMToIZiPSxToNRPog="
-                    },
-                    {
-                      "type": "Integer",
-                      "value": "50000000"
-                    }
-                  ]
-                }
-              }
-            ]
-          }
-        ]
-      }
-    }
-  },
-  {
-    "Name": "getapplicationlogasync_triggertype",
-    "Request": {
-      "jsonrpc": "2.0",
-      "method": "getapplicationlog",
-      "params": [ "0x6ea186fe714b8168ede3b78461db8945c06d867da649852352dbe7cbf1ba3724", "OnPersist" ],
-      "id": 1
-    },
-    "Response": {
-      "jsonrpc": "2.0",
-      "id": 1,
-      "result": {
-        "blockhash": "0x6ea186fe714b8168ede3b78461db8945c06d867da649852352dbe7cbf1ba3724",
-        "executions": [
```

Comment:
Ah, now I see there are even more of them. Please, do not change the API. Optimizations are OK, but we need a stable API. See #380 also.